nvflare/fuel/hci/client/config.py

```diff
@@ -39,10 +37,6 @@ def __init__(self, workspace: Workspace):
         base_pkgs = FL_PACKAGES
         module_names = FL_MODULES
 
-        custom_dir = workspace.get_client_custom_dir()
```

Comment:
Why this change? This is to add the client's local/custom to the sys path.

Reply:
Added back the custom code folder for admin tool.